src/input.cpp

```diff
@@ -237,8 +252,11 @@ bool raw_term::parse(const lexemes& l, size_t& pos, const raw_prog& prog) {
 	lexeme s = l[pos];
 	if ((neg = *l[pos][0] == L'~')) ++pos;
 	bool rel = false, noteq = false, eq = false, leq = false, gt = false,
-		bltin = false;
-	while (!wcschr(L".:,;{}|&-<", *l[pos][0])) {
+		lt = false, geq = false, bltin = false;
+	//while (!wcschr(L".:,;{}|&-<", *l[pos][0])) {
+	// D: why is '<' here? I'm guessing to detect input. Messes up LT.
+	// it's already used in direction, so just for rterm, whe/where?
+	while (!wcschr(L".:,;{}|&-", *l[pos][0])) {
```

Comment:
"->"  implication
"<->" coimplication

The - and < correspond to the first character the above lexemes that act as delimiter.